src/ffs/file.ts

```diff
@@ -28,21 +19,19 @@ export async function catRaw(cid: CID): Promise<Uint8Array[]> {
   return chunks
 }
 
-export async function cat(cid: CID): Promise<string> {
+export async function catBuf(cid: CID): Promise<Uint8Array> {
   const raw = await catRaw(cid)
-  return Buffer.concat(raw).toString()
+  return Buffer.concat(raw)
 }
 
-export async function cidToDAGLink(cid: CID, name: string): Promise<DAGLink> {
-  const ipfs = await getIpfs()
-  const stat = await ipfs.object.stat(cid)
-  return new dagPB.DAGLink(name, stat.CumulativeSize, cid)
+export async function cat(cid: CID): Promise<string> {
```

Comment:
Not a request or anything, but just wondering, how difficult would it be to make this behave the same as the js-ipfs API and have everything return an `AsyncIterable`? Or is this only possible if there were streaming?

Reply:
yeah very manageable to do for public files. Not possible with private files until we have streaming crypto. But I was gonna to keep the APIs the same until it's supported for both

Reply:
Purrfect 🐈